build.gradle

```diff
@@ -134,7 +134,9 @@ allprojects {
             jetty                : '9.4.12.v20180830',
             mongo                : '3.6.0',
```

Comment:
Can be removed. The associated mongo-server dependency in clouditor-engine can be removed as well